java8-apifuture/src/test/java/net/javacrumbs/futureconverter/java8apifuture/ExceptionTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2014-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.javacrumbs.futureconverter.java8apifuture;
+
+import org.junit.Test;
+
+import java.util.concurrent.CompletableFuture;
+
+public class ExceptionTest {
+
+    @Test
+    public void testException() throws InterruptedException {
+        CompletableFuture<String> future = new CompletableFuture<>();
```

Comment:
Can this test ever fail?

Reply:
Hm, good thing that you spotted it, I'll fix! I blindly copy-pasted it from here, which I guess has the same issue: https://github.com/lukas-krecan/future-converter/blob/master/java8-guava/src/test/java/net/javacrumbs/futureconverter/java8guava/ExceptionTest.java#L25

Reply:
Ok, pushed a new change now, both `ApiFuture` and `ListenableFuture` are covered now.